ZuluContent/Zulu/Engines/Magic/EnchantmentDictionary.cs

```diff
@@ -137,10 +138,6 @@ public void SetFromResourceType(Type enchantmentType, int value)
             {
                 Set((FireProtection e) => e.Value = value);
             }
-            else if (enchantmentType == typeof(HolyProtection))
```

Comment:
Removing holy?

Reply:
Yeah it's not actually used, figured I'd remove it for simplicity